src/index.html

```diff
@@ -3,10 +3,45 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <title>Calendar</title>
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
```

Comment:
You don't need another `div` to center everything, you can define a `container` class for the `body`.

Reply:
I did as you said in one of the previous tasks, but for some unknown reason not this time.